### PR TITLE
Crdcdh 355 authz 001 pgu

### PR DESCRIPTION
**Pull request by vshand11**
implemented grantToken and createtempcredential APIs.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ let config = {
     //session
     session_secret: process.env.SESSION_SECRET,
     session_timeout: parseInt(process.env.SESSION_TIMEOUT_SECONDS) * 1000 || 30 * 60 * 1000,
+    token_secret: process.env.TOKEN_SECRET,
```
Comment (n2iw) [inline config.js:17]: Can we just use session_secret to save an environment variable?

File: routers/user.js [hunk 2]
```diff
@@ -1,109 +0,0 @@
-const {getCurrentTimeYYYYMMDDSS} = require("../utility/time-utility");
```
Comment (n2iw) [inline routers/user.js:1]: Is deleting this file on purpose? If so, what about the functions defined in the file?

**Review by n2iw — CHANGES_REQUESTED** [review]
Please include submodule change in this PR, after PR in the submodule has been merged.

File: routers/user.js (hunk 2 above)
Comment (vshand11) [reply routers/user.js:1]: It is not used and referenced.

**Comment by vshand11** [comment]
Not need after merged codes for grantToken and createBatch PRs

**Pull request closed without merging**